qa/ProfilingGuidelinesR4-StructureDefinitions-Zib.xml

```diff
@@ -69,19 +69,36 @@
       <path value="StructureDefinition.differential.element" />
       <constraint>
         <key value="sd-zpg-08" />
-        <severity value="error" />
+        <severity value="error"/>
         <human value="If an alias exists, mapping.map needs to be present." />
         <expression value="alias.exists() implies mapping.map.exists()" />
-        <source value="https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4" />
+        <source value="https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4"/>
+      </constraint>
+      <constraint>
+        <key value="sd-pg-02"/>
+        <human value="If mapping.map exists and the mapping is not implicit, short should exist"/>
+        <expression value="mapping.select(map.exists() and comment.contains(' (implicit, main mapping is on ').not()).anyTrue() implies short.exists()"/>
+        <source value="https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4"/>
+      </constraint>
+      <constraint>
+        <key value="sd-pg-03"/>
+        <severity value="error"/>
+        <human value="If mapping.map exists and the mapping is not implicit and the element is not the root element, definition should exist."/>
+        <expression value="(mapping.select(map.exists() and comment.contains(' (implicit, main mapping is on ').not()).anyTrue() and $this.indexOf('.') != -1) implies definition.exists()"/>
+        <source value="https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4"/>
+      </constraint>
+      <constraint>
+        <key value="sd-pg-04"/>
+        <severity value="error"/>
+        <human value="If mapping.map exists and the mapping is not implicit, alias should exist."/>
+        <expression value="mapping.select(map.exists() and comment.contains(' (implicit, main mapping is on ').not()).anyTrue() implies alias.exists()"/>
+        <source value="https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4"/>
       </constraint>
-    </element>
-    <element id="StructureDefinition.differential.element.type">
-      <path value="StructureDefinition.differential.element.type" />
       <constraint>
         <key value="sd-zpg-03" />
         <severity value="error" />
-        <human value="References to other resource profiles should be added next to the HL7  base references." />
-        <expression value="code = 'Reference' implies targetProfile.where($this.startsWith('http://hl7.org/fhir/StructureDefinition/')).exists()" />
+        <human value="References to other resource profiles should be added next to the HL7  base references. This does not apply for extensions." />

```

Comment:
This is the wrong approach. The check can simply be moved to the StructureDefinitions-Zib-Profiles profile.

Reply:
You are right. Adjusted per your feedback.